packages/toolkit/src/dao/acl/getAppPermissions.ts

```diff
@@ -1,50 +1,25 @@
-import { initWrapper } from '../utils/wrapper'
-import { AragonApp, AclPermissions } from '../../types'
-
-// TODO: Stop using wrapper
+import { connect, App, Permission } from '@aragon/connect'
```

Comment:
Here I started using `@aragon/connect` to fetch the data that the toolkit needs. We should replace the wrapper for connect everywhere.